package.json

```diff
@@ -159,7 +159,7 @@
     "jsdom": "^12.2.0",
     "jsonld": "^5.2.0",
     "jsonlint-mod": "^1.7.6",
-    "lighthouse-plugin-publisher-ads": "^1.5.6",
+    "lighthouse-plugin-publisher-ads": "https://gitpkg.now.sh/connorjclark/publisher-ads-lighthouse-plugin/lighthouse-plugin-publisher-ads?esm",
```

Comment:
i love this

Reply:
does this lock to a commit / tag? or what happens when you add more commits that aren't ready for integration?